src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4578,11 +4578,8 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_EncodinglessSyntaxTree" xml:space="preserve">
     <value>Cannot emit debug information for a source text without encoding.</value>
   </data>
-  <data name="ERR_AccessorListAndExpressionBody" xml:space="preserve">
-    <value>Properties cannot combine accessor lists with expression bodies.</value>
-  </data>
   <data name="ERR_BlockBodyAndExpressionBody" xml:space="preserve">
-    <value>Methods and accessors cannot combine block bodies with expression bodies.</value>
+    <value>Block bodies and expression bodies cannot both be provided at the same time</value>
```

Comment:
Perhaps drop `at the same time` since it implies the two can be provided separately.